libnmstate/nm/applier.py

```diff
@@ -132,6 +133,12 @@ def set_ifaces_admin_state(ifaces_desired_state, con_profiles=()):
         nmdev = device.get_device_by_name(ifname)
         if not nmdev:
             if ifname in new_ifaces and iface_desired_state['state'] == 'up':
+                master_iface_types = ovs.BRIDGE_TYPE, bond.BOND_TYPE, LB.TYPE
```

Comment:
Now that this is used at least twice here they merit a helper/constant.

But a more important point is to name this check correctly through a helper function.
Like `if _is_master_iface() and not _is_slave_iface():`

Reply:
Done.